packages/runner/src/runner.ts

```diff
@@ -1081,4 +1123,24 @@ export class Runner {
     this.logger("error", errMsg);
     throw new Error(errMsg);
   }
+
+  async _injectStakeCell(
+    txSkeleton: TransactionSkeletonType
+  ): Promise<TransactionSkeletonType> {
+    // Add stake lock dep
+    txSkeleton = txSkeleton.update("cellDeps", (cellDeps) => {
+      return cellDeps.push(this.config.deploymentConfig.stake_lock_dep);
+    });
+
+    const stakeCell: Cell = await this._queryValidStakeCell();
+    // Add stake cell input
+    txSkeleton = txSkeleton.update("inputs", (inputs) =>
+      inputs.push(stakeCell)
+    );
+    // Add stake cell output
+    txSkeleton = txSkeleton.update("outputs", (outputs) =>
+      outputs.push(stakeCell)
```

Comment:
This would not be passed the validator, the args.stake_block_number must equal to the tip block number.

Reply:
fixed.